lib/dataset-tools/augmentation/replace_parameters.js

```diff
@@ -425,7 +425,7 @@ module.exports = class ParameterReplacer {
             }
 
             // fail immediately if sampled value includes return char
-            if (sampled.includes('\n')){
+            if (sampled.includes('\n') || sampled.includes('/') || sampled.includes('\\') ){
```

Comment:
What's going on here? Sampled is a tokenized value so it cannot have `\n`. And why is `/` or `\\` problematic? We certainly want to understand words with slashes.

Reply:
For some languages, the tokenization was wrong for parameters with slashes in them. I will remove this and revisit later if the problem came up again.

Reply:
We don't split on slashes, but we do split on backslashes. At least in English. I think other languages do the same because they don't have custom rules.